w2w/w2w.py

```diff
@@ -489,6 +494,7 @@ def wrf_remove_urban(
                     south_north=i, west_east=j, land_cat=12
                 ).values
                 newluf[12, i, j] = 0.0
+        print(datetime.now() - startTime)
```

Comment:
I think this is not supposed to be in here right?

Reply:
ah, that's clumsy. These were the test statements I used to check the efficiency of that routine. I removed them now.